OpenRA.Game/Settings.cs

```diff
@@ -142,6 +142,7 @@ public class SoundSettings
 
 		public bool CashTicks = true;
 		public bool Mute = false;
+		public bool DisplayAltTexts = false;
```

Comment:
Possible workaround: to use something different than "Battlefield Control" e.g. "Server info" for these messages and in IngameChatLogic.AddChatLine adapt it like:
if (!replayCache **&& from != "Server info"**)
   Game.Sound.PlayNotification(modRules, null, "Sounds", "ChatLine", null);
(We could also display them in another place/color.)